src/pybind/mgr/cephadm/module.py

```diff
@@ -288,6 +300,7 @@ def __init__(self, *args, **kwargs):
             self.registry_url: Optional[str] = None
             self.registry_username: Optional[str] = None
             self.registry_password: Optional[str] = None
+            self.use_repo_digest = False
```

Comment:
I insist. True by default seems more safe for everybody

Reply:
:+1; agree w/ @jmolmo